3.5/README.md

```diff
@@ -244,3 +255,11 @@ docker exec -it <CONTAINER_ID> /bin/bash
 
 After you enter into the running container, your current directory is set
 to `/opt/app-root/src`, where the source code is located.
+
+
+See also
+--------
+Base Python environment Dockerfile and other sources for building the final image 
+with your Python application are available on https://github.com/sclorg/s2i-python-container.
```

Comment:
Neither. You do not need to have the sources for s2i-python to be able to build an image containing your python application. Source-to-image does all the magic for you.
The plain "Dockerfile and other sources for this container image are available on" would be better.